### PR TITLE
Change the default values of multiple tiflash configs

### PR DESCRIPTION
**Pull request by hehechen**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)
1. Add description of tiflash-proxy apply-pool-size and store-pool-size configuration. These two configurations have existed since TiDB 4.0 version, this PR is just to supplement the description of the configuration. 
2. TiFlash has [changed default values of some configurations](https://github.com/pingcap/tics/pull/3414), so the doc need to be updated.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/7962
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by solotzg — APPROVED** [review]

File: tiflash/tiflash-configuration.md [hunk 1]
```diff
@@ -208,10 +212,16 @@ delta_index_cache_size = 0
 [server]
     engine-addr = The external access address of the TiFlash coprocessor service.
 [raftstore]
+    ## The allowable number of threads in the pool that flushes Raft data to storage.
+    apply-pool-size = 4
+
+    ## The allowable number of threads that process Raft, which is the size of the Raftstore thread pool.
+    store-pool-size = 4
+
     ## Specifies the number of threads that handle snapshots.
```
Comment (shichun-0415) [inline tiflash/tiflash-configuration.md:221]:
```suggestion
    ## The number of threads that handle snapshots.
```
The modification is to unify the style.

File: tiflash/tiflash-configuration.md [hunk 2]
```diff
@@ -208,10 +212,16 @@ delta_index_cache_size = 0
...
-    ## The default number is 2.
+    ## The default number is 4.
```
Comment (shichun-0415) [inline tiflash/tiflash-configuration.md:222]:
```suggestion
    ## The default number is `4`.
```

File: tiflash/tiflash-configuration.md [hunk 3]
```diff
@@ -208,10 +212,16 @@ delta_index_cache_size = 0
...
     ## If you set it to 0, the multi-thread optimization is disabled.
```
Comment (shichun-0415) [inline tiflash/tiflash-configuration.md:223]:
```suggestion
    ## If you set it to `0`, the multi-thread optimization is disabled.
```

File: tiflash/tiflash-configuration.md [hunk 4]
```diff
@@ -208,10 +212,16 @@ delta_index_cache_size = 0
...
-    snap-handle-pool-size = 2
+    snap-handle-pool-size = 4
 
     ## Specifies the shortest interval at which Raft store persists WAL.
```
Comment (shichun-0415) [inline tiflash/tiflash-configuration.md:226]:
```suggestion
    ## The shortest interval at which Raft store persists WAL.
```

**Review by shichun-0415 — APPROVED** [review]
LGTM

**Comment by shichun-0415** [comment]
@hehechen Could you please synchronize the change in the default value of `status-addr` and `addr` to the English version? Thanks.

File: tiflash/tiflash-configuration.md [hunk 5]
```diff
@@ -136,28 +136,32 @@ delta_index_cache_size = 0
...
-    errorlog = The TiFlash error log path.
-    size = The size of a single log file.
-    count = The maximum number of log files to save.
+    ## log level (available options: trace, debug, information, warning, error). The default value is `debug`.
+    level = debug
+    log = TiFlash log path
+    errorlog = TiFlash error log path
+    ## Size of a single log file. The default value is `100M`.
+    size = "100M"
+    ## Maximum number of log files to save. The default value is `10`.
```
Comment (qiancai) [inline tiflash/tiflash-configuration.md:155]:
```suggestion
    ## Maximum number of log files to save. The default value is 10.
```

File: tiflash/tiflash-configuration.md [hunk 6]
```diff
@@ -208,19 +212,25 @@ delta_index_cache_size = 0
...
     ## You can properly increase the latency to reduce IOPS usage.
     ## The default value is "4ms".
     ## If you set it to 0ms, the optimization is disabled.
     store-batch-retry-recv-timeout = "4ms"
 [security]
     ## New in v5.0. This configuration item enables or disables log redaction.
-    ## If the configuration value is set to true,
+    ## If the configuration value is set to "true",
     ## all user data in the log will be replaced by ?. The default value is false.
     redact-info-log = false
```
Comment (qiancai) [inline tiflash/tiflash-configuration.md:235]: "redact-info-log" is not in the Chinese doc. Please check if it should be added.

Comment (shichun-0415) [reply tiflash/tiflash-configuration.md:235]: will fix in another PR.

**Review by qiancai — APPROVED** [review]
LGTM

**Pull request merged**